src/main/java/com/conveyal/r5/analyst/cluster/AnalysisTask.java

```diff
@@ -48,26 +46,18 @@
     /** A unique identifier for this task assigned by the queue/broker system. */
     public int taskId;
 
-    /** Save results to a bucket.  If blank, the response to this task will be via the default channel (broker for
-     *  single-point requests, queue for regional requests).  Not yet compatible with UI.
+    /**
+     * Whether to save results on S3. If null or empty, the response to this task will be via the
```

Comment:
Is this comment still correct?  Results will be via the default channel if `makeStaticSite` is false, rather than null or empty.

Reply:
The comment is definitely wrong and I'll update it. This one is tricky though because I'm aiming for a non-dichotomous situation where you can set this flag to "true" and whatever results are produced by each task, which may be travel time or accessibility results or both, will be saved to S3.